cmd/moby/main.go

```diff
@@ -7,6 +7,7 @@ import (
 	"path/filepath"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/homedir"
```

Comment:
Not going to vendor code from docker/docker to do trivial things, as we are removing all the docker dependencies from this package, and also the code in docker/docker/pkg is mostly not worth using. You do not need anything other than `os.Getenv("HOME")` on Unix or `os.Getenv("USERPROFILE")` on Windows.